### ISSUE TITLE
Failures on [cfe] Compute correct type for read of late lowered variables

### ISSUE THREAD

**Issue by johnniwinther** [dart-lang/sdk#40311]
There are new test failures on CL [[cfe] Compute correct type for read of late lowered variables](https://dart-review.googlesource.com/c/sdk/+/133102).

The tests
```
dartdevc_native/debugger/debugger_test RuntimeError (expected Pass)
lib/collection/failing_list_test RuntimeError (expected Pass)
```
are failing on configurations
```
dartdevk-strong-linux-release-chrome
dartdevk-weak-linux-release-chrome
```

**Comment by fishythefish** [comment]
Nick, feel free to close if these issues have already been addressed.

**Comment by nshahan** [comment]
These tests were fixed in e352744fa4a8c97c384694861f3ffd8e1dceabef and all are passing now.
